src/coffee/utils/array-sync.coffee

```diff
@@ -46,8 +46,7 @@ angular.module('uiGmapgoogle-maps').factory 'uiGmaparray-sync', [
           set_at: (index) ->
             return if isSetFromScope #important to avoid cyclic forever change loop watch to map event change and back
             value = mapArray.getAt(index)
-            return  unless value
-            return  if not value.lng or not value.lat
+            return unless value and value.lng and value.lat
```

Comment:
Actually this would have been best `return unless value?.lng and value.lat` then one check for existence and it satisfies me and u.